### DISCUSSION TITLE
Changes to layout template won't display

### DISCUSSION THREAD

**Discussion by jayrdi** [statamic/cms#6058] [Q&A] [answered]
I am following the statamic tutorial videos and have made an edit to `layout.antlers.html` as per the video:

![image](https://user-images.githubusercontent.com/7912545/169055880-87e407e3-06df-45ba-8385-59674c318c6e.png)

...but this makes no change to my statamic test site; the background is still white. Does anyone know what I'm doing wrong?

Thanks

**Comment by jackmcdade** [comment, marked as answer]
Run `npm run watch` so Tailwind gets recompiled when you change the classes you're using. More info about Tailwind's JIT mode here: https://tailwindcss.com/blog/just-in-time-the-next-generation-of-tailwind-css

**Reply by jayrdi** [reply]
Thanks, actually I needed to clear my browser cache, ugh
